src/modules/addToDo.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+
+// Add to Do
+import { todos } from '../index.js';
+
+const addToList = document.getElementById('addToList');
+
+const addToDo = () => {
+  const task = addToList.value;
+  const num = 0;
+
+  todos.push({
+    description: task,
+    completed: false,
+    index: num,
+  });
+  addToList.value = '';
+};
```

Comment:
- [ ] @HunterX-7 great work implementing the add function, however as per the project requirement your index value should be equal to the length of the new array.
![image](https://user-images.githubusercontent.com/61408860/206840875-429884c9-0499-40ea-974f-e1ddf62b294d.png)